config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # customer routes
+  get 'customers/:id', to: 'customers#show'
+  post 'customers/:id/items', to: 'customer_items#create'
```

Comment:
I like this choice of a route for creating a new item tied to a specific customer.
Your routes look good and are following restful convention.